tfhe/src/integer/server_key/radix_parallel/tests_cases_unsigned.rs

```diff
@@ -3911,10 +3923,12 @@ where
     let (cks, sks) = KEY_CACHE.get_from_params(param, IntegerKeyKind::Radix);
     let sks = Arc::new(sks);
 
-    let cks = RadixClientKey::from((cks, NB_CTXT));
+    let nb_ctxt = 4;
```

Comment:
nb_ctxt = 4.max(NB_CTXT)

maybe ? and indicate we need at least 4 blocks in a comment (I guess that's why this was changed to 4)